apps/web/src/common/modules/navigations/gnb/modules/gnb-menu/modules/dashboard-recent-favorite/modules/GNBDashboardMenu.vue

```diff
@@ -130,6 +130,7 @@ export default defineComponent({
         };
 
         (async () => {
+            // CAUTION: If GNBDashboardMenu is deprecated, you need to add a request to receive a dashboard list in "GNBFavorite.vue".
```

Comment:
question: This caution comment want to warn not to delete it?

Reply:
Yes, I was going to make warning sign.